src/main/kotlin/com/javacafe/realtimewinnmin/presentation/controller/NewsController.kt

```diff
@@ -0,0 +1,120 @@
+package com.javacafe.realtimewinnmin.presentation.controller
+
+import com.javacafe.realtimewinnmin.application.dto.NewsCreateRequest
+import com.javacafe.realtimewinnmin.application.dto.NewsListResponse
+import com.javacafe.realtimewinnmin.application.dto.NewsResponse
+import com.javacafe.realtimewinnmin.application.dto.NewsSearchRequest
+import com.javacafe.realtimewinnmin.application.service.NewsService
+import com.javacafe.realtimewinnmin.common.dto.ApiResponse
+import com.javacafe.realtimewinnmin.common.exception.ExceptionCode
+import com.javacafe.realtimewinnmin.common.exception.GlobalException
+import org.slf4j.LoggerFactory
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/api/news")
+class NewsController(
+    private val newsService: NewsService
+) {
+    private val logger = LoggerFactory.getLogger(NewsController::class.java)
```

Comment:
Kotlin Logger가 더 다양한 기능을 많이 지원해욤욤

```suggestion
      private val logger = KotlinLogging.logger { }
```

p99)

Reply:
바로 차용하겠습니다!!